### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by tpoisot** [samu/git-diff-details#5]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.activate (/Users/tpoisot/.atom/packages/git-diff-details/lib/main.coffee:7:9)
```

**Comment by samu** [comment]
Thanks for the hint and the detailed description. I'm quite busy right now and will try to fix this in the coming week.

**Comment by samu** [comment]
fixed with #19
